maestro-orchestra/src/main/java/maestro/orchestra/Orchestra.kt

```diff
@@ -382,6 +382,9 @@ class Orchestra(
             throw MaestroException.CloudApiKeyNotAvailable("`MAESTRO_CLOUD_API_KEY` is not available. Did you export MAESTRO_CLOUD_API_KEY?")
         }
 
+        val maestroCommand = MaestroCommand(command)
```

Comment:
Why are we creating another copy object of maestro command isn't this same thing passed from executeCommand function?